recipe/build.sh

```diff
@@ -102,9 +102,11 @@ cmake .. -LAH                                                             \
     -DWITH_FFMPEG=1                                                       \
     -DWITH_MATLAB=0                                                       \
     -DWITH_VTK=0                                                          \
+    -DWITH_QT=5                                                           \
     -DWITH_GPHOTO2=0                                                      \
     -DINSTALL_C_EXAMPLES=0                                                \
     -DOPENCV_EXTRA_MODULES_PATH="../opencv_contrib-$PKG_VERSION/modules"  \
+    -DBUILD_opencv_cvv=OFF                                                \
```

Comment:
Yes, OS X works as of right now because Open CV picks up Cocoa. I am unsure if it will work with QT5, but @jakirkham suggested we should strive for a uniform build across platforms. @patricksnape do you have a way to trigger this build on travis and get the package into an anaconda repo? Is there some way for me to set up my fork of the feedstock and hook it up to a personal anaconda repo so I can get test builds to try locally?